--- installer.conf	2018-01-15 18:12:58.000000000 +0800
+++ installernew.conf	2018-01-15 18:12:16.000000000 +0800
@@ -1,6 +1,6 @@
 # Accton as5712_54x ONIE installer configuration
 
-description="Accton, as5712"
+description="Lanner, NCB-4210"
 
 mass_dev="storage-not-found"
 discover_mass_storage()
@@ -18,7 +18,9 @@
 # Default ONIE block device
 install_device_platform()
 {
-    discover_mass_storage "target1:0:0" 
+    discover_mass_storage "target2:0:0" 
+    discover_mass_storage "target3:0:0" 
+    discover_mass_storage "target4:0:0" 
     ret=$?
     echo "$mass_dev"
     return $ret
